solutions/fully-configurable/variables.tf

```diff
@@ -237,7 +239,7 @@ variable "ibmcloud_kms_api_key" {
 
 variable "enable_collecting_failed_events" {
   type        = bool
-  description = "Set to true to enable Cloud Object Storage integration. If true a Cloud Object Storage instance to store failed events in should also be passed using variable `existing_cos_instance_crn`. For more info see https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-cfe-integrations."
+  description = "Set to true to enable Cloud Object Storage integration. If true a Cloud Object Storage instance to store failed events in should also be passed using variable `existing_cos_instance_crn`. For more info, [Learn more](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-cfe-integrations)."
```

Comment:
`If true a Cloud....` -> sentence formation is not correct.
Again, `For more information, Learn more` does not sound good.